packages/fx-core/src/plugins/resource/bot/v2/enum.ts

```diff
@@ -11,3 +11,7 @@ export enum Runtime {
   Node = "node",
   Dotnet = "csharp",
 }
+
+export enum BicepModules {
+  Functions = "botFunction",
```

Comment:
Nit: Consistency - `Function + s` vs `botFunction`

Reply:
Functions is the official naming while botFunction is already used in bicep. For backward compatibility we should not change the value.